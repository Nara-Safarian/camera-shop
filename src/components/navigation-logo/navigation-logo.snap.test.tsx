import { render } from '@testing-library/react';
import NavigationLogo from './navigation-logo';
import { BrowserRouter } from 'react-router-dom';

test('NavigationLogo snapshot should be rendered correctly', () => {
  const {container} = render(
    <BrowserRouter>
      <NavigationLogo />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
