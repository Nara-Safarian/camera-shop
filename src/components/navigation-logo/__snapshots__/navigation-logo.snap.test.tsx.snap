// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLogo snapshot should be rendered correctly 1`] = `
<div>
  <a
    aria-label="Переход на главную"
    class="header__logo"
    href="/"
  >
    <svg
      aria-hidden="true"
      height="36"
      width="100"
    >
      <use
        xlink:href="#icon-logo"
      />
    </svg>
  </a>
</div>
`;
